client/os.cpp

```diff
@@ -424,9 +464,10 @@ namespace tec {
 		if (action == GLFW_PRESS) {
 			mbtn_event->action = MouseBtnEvent::DOWN;
 		}
-		else {
+		else if(action == GLFW_RELEASE) {
 			mbtn_event->action = MouseBtnEvent::UP;
 		}
+		else return;
```

Comment:
Without the bracing this is a bit hard to see since the other branches use braces.